pkg/cmd/cli/config/loader.go

```diff
@@ -14,19 +14,19 @@ import (
 )
 
 const (
-	OpenShiftConfigPathEnvVar      = "OPENSHIFTCONFIG"
+	OpenShiftConfigPathEnvVar      = "KUBECONFIG"
 	OpenShiftConfigFlagName        = "config"
-	OpenShiftConfigHomeDir         = ".config/openshift"
+	OpenShiftConfigHomeDir         = ".kube"
 	OpenShiftConfigHomeFileName    = "config"
 	OpenShiftConfigHomeDirFileName = OpenShiftConfigHomeDir + "/" + OpenShiftConfigHomeFileName
 )
 
-var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), ".config/openshift/.config")
+var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), ".kube/.config")
```

Comment:
`.kube/.config` was never a path anyone used as far as I know, can we remove this? fine in a follow-up if you like